src/component/Allproperties/Allproperties.jsx

```diff
@@ -1,279 +1,20 @@
-    name: "John Dorf",
-    updated: "2 weeks ago",
-    property: "Hammond Loft Property",
-    location: "Victoria Island",
-    sale: "sale",
-    more: "more info",
-    bed: "3bed",
-    bath: "3",
-    square: "1,878 sqft",
-  },
-
-  {
-    id: 10,
-    title: "duplex",
-    image: "/images/xwork12.jpg",
-    price: "#30,000,000",
-    name: "John Dorf",
-    updated: "2 weeks ago",
-    property: "Hammond Loft Property",
-    location: "Victoria Island",
-    sale: "sale",
-    more: "more info",
-    bed: "3bed",
-    bath: "3",
-    square: "1,878 sqft",
-  },
-
-  {
-    id: 11,
-    title: "duplex",
-    image: "/images/xwork13.jpg",
-    price: "#30,000,000",
-    name: "John Dorf",
-    updated: "2 weeks ago",
-    property: "Hammond Loft Property",
-    location: "Victoria Island",
-    sale: "sale",
-    more: "more info",
-    bed: "3bed",
-    bath: "3",
-    square: "1,878 sqft",
-  },
-
-  {
-    id: 12,
-    title: "bungalow",
-    image: "/images/xwork14.jpg",
-    price: "#30,000,000",
-    name: "John Dorf",
-    updated: "2 weeks ago",
-    property: "Hammond Loft Property",
-    location: "Victoria Island",
-    sale: "sale",
-    more: "more info",
-    bed: "3bed",
-    bath: "3",
-    square: "1,878 sqft",
-  },
-
-  {
-    id: 13,
-    title: "duplex",
-    image: "/images/xwork15.jpg",
-    price: "#30,000,000",
-    name: "John Dorf",
-    updated: "2 weeks ago",
-    property: "Hammond Loft Property",
-    location: "Victoria Island",
-    sale: "sale",
-    more: "more info",
-    bed: "3bed",
-    bath: "3",
-    square: "1,878 sqft",
-  },
-
-  {
-    id: 14,
-    title: "duplex",
-    image: "/images/xwork9.jpg",
-    price: "#30,000,000",
-    name: "John Dorf",
-    updated: "2 weeks ago",
-    property: "Hammond Loft Property",
-    location: "Victoria Island",
-    sale: "sale",
-    more: "more info",
-    bed: "3bed",
-    bath: "3",
-    square: "1,878 sqft",
-  },
-
-  {
-    id: 15,
-    title: "bungalow",
-    image: "/images/xwork7.jpg",
-    price: "#30,000,000",
-    name: "John Dorf",
-    updated: "2 weeks ago",
-    property: "Hammond Loft Property",
-    location: "Victoria Island",
-    sale: "sale",
-    more: "more info",
-    bed: "3bed",
-    bath: "3",
-    square: "1,878 sqft",
-  },
-
-  {
-    id: 16,
-    title: "bungalow",
-    image: "/images/xwork6.jpg",
-    price: "#30,000,000",
-    name: "John Dorf",
-    updated: "2 weeks ago",
-    property: "Hammond Loft Property",
-    location: "Victoria Island",
-    sale: "sale",
-    more: "more info",
-    bed: "3bed",
-    bath: "2",
-    square: "1,878 sqft",
-  },
-];
-
-const url = "http://localhost:3006/v1/properties";
 
 export const Allproperty = () => {
-  const dispatch = useDispatch();
   const [propertydata, setProperties] = useState([]);
 
+
   const getProperties = async () => {
     try {
-      const response = await axios.get(url);
+      const response = await axios.get(`${process.env.REACT_APP_API_URL}/properties`);
```

Comment:
Lets be consistent, so here since we have save other environment variable data into a variable.... let's do same thing here.